spec/features/01_visitor_sees_list_of_disney_parks_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+# [] Visiting the `/parks` path should contain a list of disney parks.
+# [] Visiting the `/parks/1` path should show the park details for a park with the ID of 1.
+# [] Visiting the root path should display a list of all parks.
+
+feature "visitor sees a list of Disney parks" do
+  scenario "sees a list of disney parks and links to park show page" do
+    magic_kingdom = Park.create!(name: 'Magic Kingdom', address: '1180 Seven Seas Dr', city: 'Lake Buena Vista', state: 'FL', zip: '32830')
+    animal_kingdom = Park.create!(name: "Animal Kingdom", address: "2901 Osceola Pkwy", city: "Orlando", state: "FL", zip: "32830")
+
+    visit parks_path
+
+    expect(page).to have_content "Magic Kingdom"
+    expect(page).to have_link "Animal Kingdom"
```

Comment:
You might as well just have two checks for the Magical Kingdom and Animal Kingdom links. Add that expectation.